addons/website_theme_install/views/views.xml

```diff
@@ -5,21 +5,37 @@
+                <field name="display_name"/>
                 <templates>
-                    <div t-name="kanban-box">
-                        <t t-set="has_screenshot" t-value="record.image_ids.raw_value.length > 0"/>
-                        <div t-if="has_screenshot" class="o_screenshot" t-attf-style="background-image: url(web/image/#{record.image_ids.raw_value[0]});"/>
-                        <div t-if="!has_screenshot" t-attf-style="background-image: url(#{record.icon.value});"/>
-                        <div class="o_button_area">
-                            <button type="object" name="button_choose_theme" states="uninstalled" class="btn btn-primary btn-sm">Install</button>
-                            <button type="action" name="website.action_website" states="installed" class="btn btn-primary btn-sm">Install</button>
-                            <a t-if="record.url.value" class="btn btn-sm btn-primary" t-att-href="record.url.value" target="_blank">Live Preview</a>
+                    <div t-name="kanban-box" t-attf-class=" o_theme_preview #{record.state.raw_value == 'installed'? 'o_theme_installed' : ''} ">
+                        <t t-set="has_image" t-value="record.image_ids.raw_value.length > 0"/>
+                        <t t-set="has_screenshot" t-value="record.image_ids.raw_value.length > 1"/>
+                        <t t-set="image_url" t-value="has_image ? 'web/image/' + record.image_ids.raw_value[0] : record.icon.value"/>
+
+                        <div class="o_theme_preview_top">
+                            <div class="o_browser_bar"/>
+                            <div t-attf-class="#{has_screenshot? 'o_theme_screenshot' : (has_image ? 'o_theme_cover' : 'o_theme_logo')}" t-attf-style="background-image: url(#{image_url});"/>
+                            <div states="uninstalled" t-attf-class="o_button_area #{has_screenshot? 'o_has_screenshot' : ''}">
+                                <button type="object" name="button_choose_theme" class="btn btn-primary">Use this theme</button>
+                                <hr t-if="record.url.value"/>
+                                <a t-if="record.url.value" class="btn btn-default btn-sm" t-att-href="record.url.value" target="_blank">Live Preview</a>
+                            </div>
+                        </div>
+                        <div class="o_theme_preview_bottom">
+                            <h5 t-if="record.display_name.value" class="text-uppercase pull-left">
+                                <img class="pull-left mr4"  t-att-src="record.icon.value" height="16" width="16"/>
+                                <b><t t-esc="record.display_name.value.replace(('Theme'), '').replace(('theme'), '')"/></b>
+                            </h5>
+                            <h6 t-if="record.category_id.value" class="text-muted pull-right">
+                                <b><t t-esc="record.category_id.value"/></b>
+                            </h6>
```

Comment:
The `o_theme_preview_bottom` bottom element should have the `clearfix` class, otherwise it has not height.